README.md

```diff
@@ -33,52 +33,92 @@ in the x-common repository.  This describes how all the language tracks are put
-[Useful Test Failures](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures).
-Here the identifier 'observed' is used instead of actual.  That's fine.  More common are words 'got' and 'want'.
-They are clear and short.  Note
-[Useful Test Failures](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures) is part of
-[Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments).  Really we like most of the
-advice on that page.
-
-Also here in leap_test.go is a benchmark.  We throw in benchmarks because they're interesting, and because it's
-idiomatic in Go to think about performance.  There is no critical use for these though.  Usually, like this one in
-leap, they will just bench the combined time to run over all the test data rather than attempt precise timings
-on single function calls.  They are useful if they let the solver try a change and see a performance effect.
-
-Finally we provide the stub file `leap.go` as a starting point for solutions.
-Not all exercises need do this; this is most helpful in the early exercises for newcomers to Go.
-By convention, the stub file for an exercise with slug `exercise-slug` must be named `exercise_slug.go`.
-This is because CI needs to delete stub files to avoid conflicting definitions.
-
-## Xgo compared
-
-We do a few things differently than the other language tracks.  In Go we generally have all tests enabled and do
-not ask the solver to edit the test program, to enable progressive tests for example.  `Testing.Fatal`, as seen
-in leap_test.go, will stop tests at the first problem encountered so the solver is not faced with too many errors
-all at once.
-
-We like errors in Go.  It's not idiomatic Go to ignore invalid data or have undefined behavior.  Sometimes our
-Go tests require an error return where other language tracks don't.
+Let's walk through the first exercise, *hello-world*. Navigate into the *hello-world*
+directory, you'll see there are three files there; *example.go*, *hello_test.go*,
+and *hello_world.go*.
+
+```sh
+~/exercism/go/hello-world
+$ tree
+.
+├── example.go
+├── hello_test.go
+└── hello_world.go
+
+0 directories, 3 files
+```
+
+### Example solutions
+
+*example.go* is a reference solution.  It is a valid solution that [Travis](https://travis-ci.org/exercism/xgo),
+the CI (continuous integration) service, can run tests against.  Solvers generally
+will not see it though.  Files with *"example"* in the file name are skipped by
+the `exercism fetch` command.  Because of this, there is less need for this code
+to be a model of style, expression and readability, or to use the best algorithm.
+Examples can be plain, simple, concise, even naïve, as long as they are correct.
+The test file though, is fetched for the solver and deserves attention for consistency
+and appearance.
+
+### Tests
+
+*hello_test.go* uses a data-driven test. Test cases are defined as data, then a
```

Comment:
It's very hard to say whether this is true of `hello_test.go` since it only tests a single case (since the function always returns `Hello, World!". It's true that most exercises use this though. Perhaps we can explain about the other exercises perhaps?